src/OpenLoco/src/Windows/ScenarioOptions.cpp

```diff
@@ -953,7 +953,7 @@ namespace OpenLoco::Ui::Windows::ScenarioOptions
         // 0x0046E306
         static uint32_t getLoanSizeInCurrency()
         {
-            uint64_t loanSizeInCurrency = Economy::getInflationAdjustedCost(CompanyManager::getStartingLoanSize(), 0, 8) / 100 * 100;
+            int32_t loanSizeInCurrency = Economy::getInflationAdjustedCost(CompanyManager::getStartingLoanSize(), 0, 8) / 100 * 100;
```

Comment:
I'm a little wary of this change, tbh, particularly as the sign changes. Could we double-check this?

Reply:
afaik this is correct. This has been wrong for a little while.

Reply:
`getInflationAdjustedCost` returns `currency32_t` which is `int32_t`, and a `/ 100 * 100` cannot possibly make it overflow. actually I will change this to `currency32_t here as well`